lang/python/python-yaml/Makefile

```diff
@@ -38,7 +38,11 @@ define Package/python3-yaml/description
   PyYAML is a YAML parser and emitter for the Python programming language.
 endef
 
-PYTHON3_PKG_BUILD_VARS:=PYYAML_FORCE_LIBYAML=1
+PYTHON3_PKG_BUILD_VARS:= \
+	PYYAML_FORCE_LIBYAML=1
```

Comment:
This could be left as it is, right? No need to touch it.

Reply:
according to your wishes, I have returned it according to your wishes. actually I don't want to fix this but I have a sense of responsibility because the cython update caused this problem. Thank you also for observing the closed pull request. maybe this is my last pull request thanks

Reply:
Since I got notification and see that you opened 2 PRs for the same thing, then you could expect the response.

Reply:
if this works I will change the message of the changes I made